src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -235,40 +235,54 @@ private void setupKeyboardShortcuts() {
                 KeyPress.setLastKeyPressedCodeAndTime(event.getCode());
             }
             if (NEW_COMMENT.match(event)) {
-                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else {
+                TurboIssue selected = getCurrentlySelectedIssue();
+                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())
+                    && selected.isPullRequest()) { // goto PR discussion tab
+
+                    ui.getBrowserComponent().openPrTab(selected, DISCUSSION_TAB);
+//                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                } else { // focus new comment textbox
                     if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
-                        openPageOfCurrentlySelectedIssue();
-                    } else if (!ui.getBrowserComponent().isCurrentUrlDiscussion()) {
+                        openTurboIssuePage(getCurrentlySelectedIssue());
```

Comment:
`openIssuePage` would be better. Also, you have `selected` variable in this scope.